fake-manage-recalls-api/stubs/__files/get-recall-empty.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recallId": "6e3ab21c-5717-4562-b3fc-2c963f66afa6",
+  "nomsNumber": "123",
```

Comment:
Why is the nomsNumber here 123? Rather than A1234AA? Think this is breaking something on api side